### PR TITLE
website: add docs to website

### PR DESCRIPTION
**Pull request by MSarmadQadeer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+---
+title: Simplex Docs
+ignoreForWeb: true
+---
```
Comment (epoberezkin) [inline README.md:4]: we should special-case it

Comment (epoberezkin) [reply README.md:4]: ... not good to have front matter in the main file

File: README.md [hunk 2]
```diff
@@ -6,11 +10,11 @@
...
 
-<img src="images/simplex-chat-logo.svg" alt="SimpleX logo" width="100%">
+<img src="/images/simplex-chat-logo.svg" alt="SimpleX logo" width="100%">
 
 # SimpleX - the first messaging platform that has no user identifiers of any kind - 100% private by design!
 
-[<img src="./images/trail-of-bits.jpg" height="100">](http://simplex.chat/blog/20221108-simplex-chat-v4.2-security-audit-new-website.html) &nbsp;&nbsp;&nbsp; [<img src="./images/privacy-guides.jpg" height="80">](https://www.privacyguides.org/en/real-time-communication/#simplex-chat) &nbsp;&nbsp;&nbsp; [<img src="./images/kuketz-blog.jpg" height="80">](https://www.kuketz-blog.de/simplex-eindruecke-vom-messenger-ohne-identifier/)
+[<img src="/images/trail-of-bits.jpg" height="100">](http://simplex.chat/blog/20221108-simplex-chat-v4.2-security-audit-new-website.html) &nbsp;&nbsp;&nbsp; [<img src="/images/privacy-guides.jpg" height="80">](https://www.privacyguides.org/en/real-time-communication/#simplex-chat) &nbsp;&nbsp;&nbsp; [<img src="/images/kuketz-blog.jpg" height="80">](https://www.kuketz-blog.de/simplex-eindruecke-vom-messenger-ohne-identifier/)
```
Comment (epoberezkin) [inline README.md:17]: can these be reverted if it's not needed for the web?

Comment (MSarmadQadeer) [reply README.md:17]: yes these can be reverted

File: README.md (hunk 1 above)
Comment (MSarmadQadeer) [reply README.md:4]: Ok I'm removing it now

**Pull request merged**